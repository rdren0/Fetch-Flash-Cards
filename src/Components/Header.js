import React, { Component } from 'react';

class Header extends Component {
  constructor(props){
    super(props);
    this.state = {}
  }

render() {
    return (
      <div className="header">
         <h1>Make Fetch Happen</h1>
          <h3> Fetch/Networks/Servers</h3>
      </div>);
    }
  
}


export default Header;
