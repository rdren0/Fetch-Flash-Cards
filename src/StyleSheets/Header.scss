h1{
  font-size: 50px;
  color: black;
  margin-top: 5px;
  margin-bottom: 10px;
}

h3{
  color: black;
}

.header{
margin: 0px;

}