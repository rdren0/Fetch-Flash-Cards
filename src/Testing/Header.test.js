import Header from '../Components/Header';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });



describe('Header', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <Header/>
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});