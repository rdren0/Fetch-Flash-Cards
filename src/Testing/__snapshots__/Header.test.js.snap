// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<div
  className="header"
>
  <h1>
    Make Fetch Happen
  </h1>
  <h3>
     Fetch/Networks/Servers
  </h3>
</div>
`;
